sandbox/embeds.html.example

```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
```

Comment:
this file will get autocopied into `sandbox/embeds.html` on the first `npm start` run by the user.
It has 18 different video.js embeds grouped by whether data-setup is used or not, how the source is added, and the 3 different embed types: video, video+player div wrapper, just div.